src/main/webapp/app/overview/courses.component.ts

```diff
@@ -111,6 +126,33 @@ export class CoursesComponent implements OnInit, OnDestroy {
         }
     }
 
+    /**
+     * Sets the course for the next upcoming exam and returns the next upcoming exam or undefined
+     */
+    get nextRelevantExam(): Exam | undefined {
+        let relevantExam: Exam | undefined = undefined;
+        if (this.nextRelevantExams) {
+            if (this.nextRelevantExams.length === 0) {
+                return undefined;
+            } else if (this.nextRelevantExams.length === 1) {
+                relevantExam = this.nextRelevantExams[0];
+            } else {
+                relevantExam = this.nextRelevantExams.sort((a, b) => {
+                    return moment(a.startDate).valueOf() - moment(b.startDate).valueOf();
```

Comment:
Could you maybe change this to use `isBefore` / `isAfter`. It would look a lot cleaner. (Even tough the logic is sometimes a bit tricky)

Reply:
@filip-gregurevic The sorting functionality of arrays does not work with booleans, therefore I would leave it how it is 😄